### PR TITLE
Update dependencies like its mid july 2023

### PR DESCRIPTION
**Pull request by paazmaya**
How to test this fully?

### CODE REVIEW

**Comment by pashidlos** [comment]
I used to utilise example repos for testing new version e2e also upgrading it there

File: package.json [hunk 1]
```diff
@@ -27,16 +27,16 @@
     "node": ">=16.16.0"
   },
   "dependencies": {
-    "@visual-regression-tracker/sdk-js": "5.6.0"
+    "@visual-regression-tracker/sdk-js": "5.7.0"
```
Comment (pashidlos) [inline package.json:30]: this is auto incremented

Comment (paazmaya) [reply package.json:30]: What do you mean by it? This version was the latest available at npm registry...

Comment (pashidlos) [reply package.json:30]: not relevant

**Comment by paazmaya** [comment]
```
(uncaught exception)CypressError: Cypress detected that you returned a promise from a command while also invoking one or more cy commands in that promise.

The command that returned the promise was:

> cy.screenshot()

The cy command you invoked inside the promise was:

> cy.task()

Because Cypress commands are already promise-like, you don't need to wrap them or return your own promise.

Cypress will resolve your command with whatever the final Cypress command yields.

The reason this is an error instead of a warning is because Cypress internally queues commands serially whereas Promises execute as soon as they are invoked. Attempting to reconcile this would prevent Cypress from ever resolving.
```

This is the current status.

**Review by pashidlos — APPROVED** [review]
LGTM

**Pull request merged**